### ISSUE TITLE
Docker-ize project

### ISSUE THREAD

**Issue by JeffreyAngell** [earlyprogrammer/CLUFFlink#1]
We should be able to have this running with docker. This will prevent us from having to install a webserver for working on these files. Also, it would make development easier on windows, if it's required.

**Comment by earlyprogrammer** [comment]
I'm not very familiar with docker, but isn't it just another local webserver?
Will it make developing easier than just using python HTTP.server?

**Comment by JeffreyAngell** [comment]
Not exactly. It's a container, which ends up being a lightweight vm. There's a few reasons we might want to do this, and I'm still working on exactly how to do it, but I'll assign it to myself.
